stock_move_purchase_price_update/readme/USAGE.md

```diff
@@ -0,0 +1,8 @@
+- Go to Sales \> Product \> Products and edit/create one
```

Comment:
Why is it necessary to go to the Sales menu? If a user doesn't have the group, it can be done in the Purchase menu.
```suggestion
- Go to Purchase \> Product \> Products and edit/create one
```

Reply:
Changes done